xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties":{
+    "facility": {
+      "type": "string",
+      "description": "Facility Name (NSLS-II, APS, etc)"
```

Comment:
These should probably be verified against a pre-defined list, so we don't get nsls2, nslsii, nslsll,nsls-ii,NSLSII and so on.  Is that possible?